composer/trainer/trainer.py

```diff
@@ -791,25 +793,16 @@ def __init__(
                 DeprecationWarning((f"Using the 'grad_clip_norm' field in Trainer is deprecated. Please use"
                                     'the GradientClipping Algorithm in composer.algorithms.gradient_clipping.')))
 
-            print_warning = False
             if algorithms is not None:
-                if isinstance(algorithms, Sequence):
-                    if any([isinstance(algo, GradientClipping) for algo in algorithms]):
-                        print_warning = True
-                elif isinstance(algorithms, GradientClipping):
-                    print_warning = True
-
-                if print_warning:
+                if any(isinstance(alg, GradientClipping) for alg in algorithms):
                     warnings.warn(
-                        RuntimeWarning(
+                        UserWarning(
                             f'The GradientClipping algorithm is already specified. Ignoring grad_clip_norm={grad_clip_norm}'
                         ))
-
-            else:
-                if algorithms is not None:
-                    algorithms.append(GradientClipping(clipping_type='norm', clipping_threshold=grad_clip_norm))
                 else:
-                    algorithms = [GradientClipping(clipping_type='norm', clipping_threshold=grad_clip_norm)]
+                    algorithms.append(GradientClipping(clipping_type='norm', clipping_threshold=grad_clip_norm))
+            else:
+                algorithms = [GradientClipping(clipping_type='norm', clipping_threshold=grad_clip_norm)]
```

Comment:
I think this can be removed -- don't think this code path will be taken (see the above comment)

Reply:
done